source/scripts/ContentScript/inject/stargazerProvider.txt.ts

```diff
@@ -179,30 +185,28 @@ const allPrivider = provider('ethereum');
 
 window.stargazer = {
   evtRegMap: {},
-  get version(){
```

Comment:
Don't we use `window.stargazer.version` in the Lattice codebase?

Reply:
yeap we do and it will be still defined in https://github.com/StardustCollective/stargazer-wallet-ext/blob/b90ab6c9af54f0b3c66d1c1998abe516a833bd68/source/scripts/InjectedScript/stargazerWalletProvider.ts#L11